__tests__/app.test.js

```diff
@@ -145,6 +145,33 @@ describe('/api/articles', () => {
             expect(response.body.msg).toBe('Bad request');
         });
     })
+    test('GET:200 accept a valid topic query, which has articles ', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toHaveLength(12)
+            body.articles.forEach((article) => {
+                expect(article.topic).toBe('mitch')              
+            })
+        })
+    })
+    test('GET:200 accept a valid topic query which has no articles ', () => {
+        return request(app)
+        .get('/api/articles?topic=paper')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.msg).toBe('No articles found');
+        });
```

Comment:
Instead of sending a message, send the empty array. If it's an okay message from the 200 then sending an error type message doesn't really match. You can use the empty array in the front end to display a message if you like